etna/core/models/basepage.py

```diff
@@ -146,45 +145,29 @@ def get_datalayer_data(self, request: HttpRequest) -> Dict[str, Any]:
         data.update(customDimension3=self._meta.verbose_name)
         return data
 
+    default_api_fields = [
+        APIField("id"),
+        APIField("title"),
+        APIField("url"),
+        APIField("full_url"),
+        APIField("type_label"),
+        APIField("teaser_text"),
+    ]
+
     api_fields = [
         APIField("type_label"),
-        APIField("teaser_image"),
         APIField("teaser_text"),
         APIField(
```

Comment:
Are we able to get a `teaser_image_alt_text` as a field as well?

Reply:
_For now_, we can just have these as decorative images but I think this is a good example of when the `alt` attribute needs to go in the image model because we don't have the ability to set it right now.

The image is attached to the page and we are just linking the page so there's no opportunity to add alternative text. 🥲

Another thing to raise with content designers and see what they want to do...